src/Cake.Web/App_Data/addins.xml

```diff
@@ -364,4 +364,15 @@
     <Description>Cake addin for compiling client code and API definitions (including Swagger) using the NSwag toolchain</Description>
     <Categories>Swagger</Categories>
   </Addin>
+  <Addin>
+    <Name>Cake.Plist</Name>
+    <NuGet Id="Cake.Plist">
+      <Filter>/**/Cake.Plist.dll</Filter>
+      <Filter>/**/Cake.Plist.xml</Filter>
+    </NuGet>
+    <Repository>https://github.com/reicheltp/Cake.Plist</Repository>
+    <Author>Paul Reichelt</Author>
+    <Description>Cake addin for deserializing, modifying and serializing Apple macOS/iOS xml plists directly from .NET source code.</Description>
```

Comment:
OSX is now macOS :trollface: :-1: (god damn it apple)

http://www.theverge.com/2016/6/22/11997918/apple-macos-sierra-preview-siri-icloud-drive